test/scapy/layers/bluetooth.uts

```diff
@@ -365,3 +381,6 @@ assert r == b'\rscapy\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
 
 p = SM_Hdr(r)
 assert SM_DHKey_Check in p and p.dhkey_check[:5] == b"scapy"
+
```

Comment:
These spaces looks useless.

Reply:
I fixed it! I need your approval

Reply:
I got "Some checks were not successful"

I only deleted that blank lines...
So it isn't related to my pull request..

I request on more approval,,, thanks..

Reply:
I need your approval..!